streamer/src/nonblocking/quic.rs

```diff
@@ -566,9 +566,12 @@ async fn handle_connection(
                         let last_update = last_update.clone();
                         tokio::spawn(async move {
                             let mut maybe_batch = None;
+                            let exit_check_interval =
+                                (wait_for_chunk_timeout_ms / 10).clamp(10, 1000);
```

Comment:
It should have no functional impact to the validator in runtime. The change still honors the timeout set by the input. It only wakes up multiple times to check if the exit condition is set so that we can quit in a more timely manner.

Reply:
The min is to guard against too small value which can wake up unnecessarily frequently and wasting CPU cycles. The max guard against waiting for too long which delay exit and cause some test failures when the timeout value is large. Within this value, the logic is to check 10 times for the set timeout if there are no data which I agree is heuristic.

Reply:
Ok, that makes sense. Could you please add a comment in the code reflecting that? It may be hard to understand the reason for the future maintainers of the code.

Reply:
Added